app/javascript/mastodon/components/account.js

```diff
@@ -21,7 +21,7 @@ export default class Account extends ImmutablePureComponent {
 
   static propTypes = {
     account: ImmutablePropTypes.map.isRequired,
-    me: PropTypes.number.isRequired,
+    me: PropTypes.string.isRequired,
```

Comment:
Were these forgotten in the other PR? Or could you rebase on top of current master so we get a cleaner diff?

Reply:
These were indeed in #5019. I'll rebase.